api/src/pcapi/alembic/versions/20211222T152126_c8ced1e097ac_add_id_pictures_stored_column.py

```diff
@@ -0,0 +1,26 @@
+"""add_id_pictures_stored_column
```

Comment:
(détail)

```suggestion
"""Add beneficiary_fraud_check.idPicturesStored
```

Ainsi, quand on fait un `alembic history`, cette description s'affiche et on sait exactement ce que fait cette migration.